TypeCobol/Compiler/Parser/CodeElementBuilder/CobolStatementsBuilder.cs

```diff
@@ -217,10 +221,8 @@ internal AllocateStatement CreateAllocateStatement(CodeElementsParser.AllocateSt
 			}
 
 			// Register call parameters (shared storage areas) information at the CodeElement level
-			var callSite = new CallSite() { CallTarget = statement.ProgramOrProgramEntryOrProcedureOrFunction.SymbolReference };
-			int parametersCount =
-				(statement.InputParameters != null ? statement.InputParameters.Count : 0)
-				+ (statement.OutputParameter != null ? 1 : 0);
+			var callSite = new CallSite() { CallTarget = statement.ProgramOrProgramEntryOrProcedureOrFunction?.SymbolReference };
+			int parametersCount = statement.InputParameters.Count + (statement.OutputParameter != null ? 1 : 0);
```

Comment:
Why has the test of nullity on `statement.InputParameters` been removed here and is still present on line 226?

Reply:
Nullity test removed, d419a3cb70b9cd23610e8f4db2e02978fff2274f.